tests/bson/bson-utcdatetime_error-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+BSON BSON\UTCDateTime #001 error
+--INI--
+date.timezone=America/Los_Angeles
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM handles parameter parsing differently"); ?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$classname = BSON_NAMESPACE . "\\UTCDateTime";
+
+throws(function() use($classname) {
+    $d = new $classname;
```

Comment:
I don't quite understand what you are trying to say, but this is how all the tests were... I just split them up.